src/types.js

```diff
@@ -36,6 +36,24 @@ export function isArray(value) {
 }
 
 /**
+ * Converts an array-like object to an array.
+ * @param {?IArrayLike<T>|string} arrayLike
+ * @return {!Array<T>}
+ * @template T
+ */
+export function toArray(arrayLike) {
+  if (!arrayLike) {
+    return [];
+  }
+  const length = arrayLike.length;
+  const array = new Array(length);
+  for (let i = 0; i < array.length; i++) {
```

Comment:
I am not modifying the length but updated to use `arrayLike.length` just in case